net/iproute2.sh

```diff
@@ -280,13 +288,25 @@ _add_route()
 		cmd="${cmd} metric ${metric}"
 	fi
 
+	# Process dev vs nodev routes
+	# Positional parameters are used for correct array handling
+	if [[ -n ${rtype} ]]; then
```

Comment:
This is a bashism, single brackets should be used instead.

Reply:
Fixed in 97a79cfd068a6cc07e9feed0ac5b71f64347ae4a, thanks.